symengine/cwrapper.h

```diff
@@ -611,6 +611,15 @@ CWRAPPER_OUTPUT_TYPE function_symbol_set(basic s, const char *c,
 CWRAPPER_OUTPUT_TYPE basic_coeff(basic c, const basic b, const basic x,
                                  const basic n);
 
+//! Wrapper for solve.h
+
+//! Solves the system of linear equations given by sys
+CWRAPPER_OUTPUT_TYPE vecbasic_linsolve(const CVecBasic *sys,
+                                       const CVecBasic *sym, CVecBasic *sol);
```

Comment:
Usually we have the output as the first argument.

Reply:
OK, thanks for the info. But there is certainly some inconsistency:
`basic_free_symbols(const basic self, CSetBasic *symbols)`

However, the output should be the first argument, and there are no exceptions to that?!

Reply:
Yes, I'd like to change `basic_free_symbols` and `basic_get_args` too. Don't know how we missed those.